repository/const.go

```diff
@@ -20,16 +20,26 @@ const (
 	MetadataColumnRow    = "METADATA$ROW_ID"
 	MetadataColumnTime   = "METADATA$TS"
 
-	queryCreateStream        = `CREATE STREAM IF NOT EXISTS %s on table %s`
-	queryCreateTrackingTable = `CREATE TABLE IF NOT EXISTS %s LIKE %s`
-	queryAddTimestampColumn  = `ALTER TABLE %s ADD COLUMN %s TIMESTAMP`
-	queryAddStringColumn     = `ALTER TABLE %s ADD COLUMN %s STRING`
-	queryAddBooleanColumn    = `ALTER TABLE %s ADD COLUMN %s BOOLEAN`
-	queryInsertInto          = `INSERT INTO %s %s`
-	queryInsertIntoColumn    = `INSERT INTO %s (%s) %s`
-	queryIsTableExist        = `SHOW TABLES LIKE '%s'`
-	queryGetMaxValue         = `SELECT MAX(%s) FROM %s`
-	queryGetPrimaryKeys      = `SHOW PRIMARY KEYS IN TABLE %s`
+	queryCreateStream         = `CREATE STREAM IF NOT EXISTS %s on table %s`
+	queryCreateTable          = `CREATE TABLE IF NOT EXISTS %s (%s, meroxa_deleted_at TIMESTAMP_LTZ, meroxa_created_at TIMESTAMP_LTZ, meroxa_updated_at TIMESTAMP_LTZ )`
+	queryCreateTemporaryTable = `CREATE TEMPORARY TABLE IF NOT EXISTS %s (%s)`
+	queryCreateStage          = `CREATE STAGE IF NOT EXISTS %s`
+	queryAddTimestampColumn   = `ALTER TABLE %s ADD COLUMN %s TIMESTAMP`
+	queryAddStringColumn      = `ALTER TABLE %s ADD COLUMN %s STRING`
+	queryAddBooleanColumn     = `ALTER TABLE %s ADD COLUMN %s BOOLEAN`
+	queryInsertInto           = `INSERT INTO %s %s`
+	queryInsertIntoColumn     = `INSERT INTO %s (%s) %s`
+	queryIsTableExist         = `SHOW TABLES LIKE '%s'`
+	queryGetMaxValue          = `SELECT MAX(%s) FROM %s`
+	queryGetPrimaryKeys       = `SHOW PRIMARY KEYS IN TABLE %s`
+	queryPutFileInStage       = `PUT file://%s @%s parallel=30;` // TODO: make parallelism configurable.
+	queryCopyInto             = `COPY INTO %s FROM @%s pattern='%s.gz' FILE_FORMAT = (TYPE = CSV FIELD_DELIMITER = ','  PARSE_HEADER = TRUE) MATCH_BY_COLUMN_NAME='CASE_INSENSITIVE' ;`
```

Comment:
I'm not sure CSV is the best type, since it's all text. Since `.Payload.After` can be JSON serialized, why not do, then use the COPY with JSON. Snowflake has a very strong JSON support. 

However, even better, we could use AVRO which matches Golang types.

Reply:
there was not a strong reason we did CSV - so I'm certainly open to changing this. I get the impression that using compression with avro or parquet can improve the performance as well, but I haven't actually compared them.